tests/beacon/test_helpers.py

```diff
@@ -873,18 +843,21 @@ def _create_slashable_vote_data_messages(params):
 
 def test_verify_slashable_vote_data_signature(privkeys,
                                               sample_beacon_state_params,
-                                              genesis_validators,
+                                              activated_genesis_validators,
+                                              genesis_balances,
                                               sample_slashable_vote_data_params):
-    sample_beacon_state_params["validator_registry"] = genesis_validators
-    state = BeaconState(**sample_beacon_state_params)
+    state = BeaconState(**sample_beacon_state_params).copy(
+        validator_registry=activated_genesis_validators,
+        validator_balances=activated_genesis_validators,
```

Comment:
should assign `genesis_balances` here

```suggestion
        validator_balances=genesis_balances,
```